etna/transforms/feature_importance.py

```diff
@@ -26,6 +30,8 @@
     CatBoostRegressor,
 ]
 
+ClusteringBased = Union[EuclideanClustering, DTWClustering]
```

Comment:
you can use https://github.com/tinkoff-ai/etna-ts/blob/master/etna/clustering/hierarchical/base.py#L25 for typings